test/it/006activityLogTest.js

```diff
@@ -27,7 +27,7 @@ describe('Activity Log Test', async () => {
     const survey = await SurveyManager.createSurvey(getContextUser(), surveyParam)
     const surveyId = Survey.getId(survey)
 
-    const surveyCreateLogs = await ActivityLogRepository.fetch(surveyId, ActivityLog.type.surveyCreate)
+    const surveyCreateLogs = await ActivityLogRepository.fetch(surveyId, true, ActivityLog.type.surveyCreate)
```

Comment:
I'm confused about what the second parameter does. Isn't fetch() typed like this?
```
export const fetch = async (surveyId, activityTypes = null, offset = 0, limit = 30, client = db) => ...
```

Reply:
the second parameter should be an array, right